docs-ref-conceptual/run-azure-cli-docker.md

```diff
@@ -48,6 +48,26 @@ To run a specific version of the Azure CLI in the Docker container, use this com
 ```bash
 docker run -it mcr.microsoft.com/azure-cli:<version>
 ```
+## Run the Azure CLI using a helper script
+
+You can create a helper script to simplify running the Azure CLI in a Docker container. Here is an example script:
+
+```bash
+#!/usr/bin/env bash
+exec docker run -it -v ${HOME}/.azure:/root/.azure -v ${HOME}/.ssh:/root/.ssh mcr.microsoft.com/azure-cli az $@
+```
+
+Save this script as `az` and make it executable:
```

Comment:
Installing the official azure-cli package is much easier than creating an `az` entry script.